README.md

```diff
@@ -73,6 +73,15 @@ archivematica_src_mcpclient_clamav_tcp_ip: "1.2.3.4"
 archivematica_src_mcpclient_clamav_tcp_port: "3310"
 ```
 
+Configure users and register pipeline
```

Comment:
Should we add a note about how/when the pipeline is automatically registered against the ss?